spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -20,110 +20,126 @@ On Ethereum, there are two types of accounts: externally owned accounts, control
 
 ### Definitions
 
+**TODO: Make this a table and add more useful definitions for used keywords throughout the spec**
+
 The IBC handler interface & IBC relayer module interface are as defined in [ICS 25](../ics-025-handler-interface) and [ICS 26](../ics-026-routing-module), respectively.
 
 ### Desired Properties
 
 - Permissionless
-- Fault containment: Interchain account must follow rules of its host chain, even in times of Byzantine behaviour by the counterparty chain (the chain that manages the account)
-- The chain that controls the account must process the results asynchronously and according to the chain's logic. The result's code should be 0 if the transaction was successful and an error code other than 0 if the transaction failed.
-- Sending and receiving transactions will be processed in an ordered channel where packets are delivered exactly in the order which they were sent.
+- Fault containment: An interchain account must follow rules of its host chain, even in times of Byzantine behavior by the counterparty chain (the chain that manages the account)
+- The chain that controls the account must process the results asynchronously and according to the chain's logic. The acknowledgment message should contain a result of an error as described in [ics-4](https://github.com/cosmos/ibc/tree/master/spec/core/ics-004-channel-and-packet-semantics#acknowledgement-envelope).
+- Sending and receiving transactions will be processed in an ordered channel where packets are delivered exactly in the order in which they were sent. 
+- Each ordered channel must only allow one interchain account to use it, otherwise, malicious users may be able to block transactions from being received. Practically, N accounts per N channels can be achieved by creating a new port for each user (owner of the interchain account) and creating a unique channel for each new registration request. Future versions of ics-27 will use partially ordered channels to allow multiple interchain accounts on the same channel to preserve account sequence ordering without being reliant on one another
```

Comment:
There are plenty of scenarios in which a module on a remote chain might be controlling multiple accounts on the destination chain, so sharing a channel adds no additional risk.

Reply:
Hey Dean, thanks for the feedback! 

Regarding multiple untrusted users (account owners) using the same channel please take a look at this comment for reference: https://github.com/cosmos/interchain-accounts/issues/22#issuecomment-820431565. 

The issue stems from multiple parties using the same ordered channel.